src/bitcoin.ts

```diff
@@ -75,36 +72,39 @@ app.use((ctx, _next) => {
 const port = process.env.SIDETREE_BITCOIN_PORT || config.port;
 
 // initialize the blockchain service and kick-off background tasks
-blockchainService.initialize()
-  .then(() => {
-    return blockchainService.startPeriodicProcessing();
-  })
-  .then(() => {
-    app.listen(port, () => {
-      console.log(`Sidetree-Bitcoin node running on port: ${port}`);
+let blockchainService: BitcoinProcessor;
+try {
+  blockchainService = new BitcoinProcessor(config);
+
+  blockchainService.initialize()
+    .then(() => {
+      app.listen(port, () => {
+        console.log(`Sidetree-Bitcoin node running on port: ${port}`);
+      });
+    })
+    .catch((error) => {
+      console.log(`Sidetree-Bitcoin node initialization failed with error ${JSON.stringify(error)}`);
     });
-  })
-  .catch((e) => {
-    console.log(`Sidetree-Bitcoin node initialization failed with error ${e}`);
-  });
+} catch (error) {
+  console.log('Is bitcoinWalletImportString valid? Consider using testnet key...');
+  console.log(BitcoinProcessor.generatePrivateKey('testnet'));
+  process.exit(1);
+}
+console.info('Sidetree bitcoin service configuration:');
+console.info(config);
 
 /**
  * Sets the koa response according to the Sidetree response object given.
  * @param response Response object fetched from request handler.
  * @param koaResponse Koa Response object to be filled
  * @param contentType Content type to be set for response, defaults to application/json
  */
-const setKoaResponse = (response: ISidetreeResponse, koaResponse: Koa.Response, contentType?: string) => {
-  koaResponse.status = SidetreeResponse.toHttpStatus(response.status);
+const setKoaResponse = (response: any, koaResponse: Koa.Response, contentType?: string) => {
+  koaResponse.status = SidetreeResponse.toHttpStatus(200);
```

Comment:
```suggestion
  koaResponse.status = 200;
```
Sanity: did you mean to do this?